packages/next-dashboard/src/components/Statistic.js

```diff
@@ -31,14 +32,29 @@ function Statistic({
   direction,
   className,
   isLoading = false,
+  positive = 'up',
   ...rest
 }: Props): R.Node {
+  let isNegative = false;
+  let isPositive = false;
+  if (positive === 'down') {
+    isPositive = direction === 'down';
+    isNegative = direction === 'up';
+  } else if (positive === 'up') {
+    isPositive = direction === 'up';
+    isNegative = direction === 'down';
+  }
```

Comment:
```js
const isPositive = (direction === positive);
const isNegative = (direction !== positive);
```
? I find it easier to reason about

Reply:
```js
const positive = null;
const isNegative = down !== positive;
IsNegatve == true; // true
```
positive set to null (or n/a) means neither will be true, aka, neutral.

Reply:
Additionally.. I think I'd name the prop `positiveDirection` or something to make it a bit more clear that it's an enum and not a boolean. To me, `positive` and `negative` sound like boolean props (so you'd do something like `<Statistic positive value={10} />` or whatever)...

Minor thing, but I figure it doesn't hurt to suggest.. :)

Reply:
```js
let isPositive = false;
let isNegative = false;
if(positive != null) {
 isPositive = (direction === positive);
 isNegative = (direction !== positive);
}
```
maybe in that case?

Reply:
Yeah, my brain has been barely functioning at all today, so nothing is named in a sane way...

_looks at branch name_